backend/qemu.pm

```diff
@@ -523,6 +523,13 @@ sub setup_tpm ($self, $arch) {
     mkdir $vmpath unless -d $vmpath;
     my $vmsock = "$vmpath/swtpm-sock";
     unless (-e $vmsock) {
+        # Before create swtpm-sock, we should make sure there is no tpm*.permall file
+        # When tpm version is 2.0, the file is tpm2-00.permall.
+        # When tpm version is 1.x, the file is tpm-00.permall.
+        # See: https://progress.opensuse.org/issues/107155
+        runcmd(('rm', "$vmpath/tpm-00.permall")) if (-e "$vmpath/tpm-00.permall");
```

Comment:
CI check failed.

at the same time, how about using "rm -rf tpm*.permall"?

Reply:
this is perl not bash

Reply:
e.g.
```
unlink glob "tpm*.permall";
```

Reply:
Modified. Thanks all.